src/main/java/org/spongepowered/api/resource/pack/PackOrder.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.resource.pack;
+
+import org.spongepowered.api.Sponge;
+
+public interface PackOrder {
```

Comment:
We're not allowing custom packs to be loaded, right? Is there any need for this? "first" and "last" only have limited value when you have more than two packs.

Reply:
Basically the thought process in Mojang's structure is the first pack will load all the assets (and additional ones mind you) and the last pack will get final say.

They are both important in their own way...granted the first pack not as much as the last.

Unsure, maybe a plugin dev would want to know what the last pack is? (and not so much the first...)

We could toss it, read the updated javadocs and this comment and see what you think. I am thinking removal as well.

Reply:
> read the updated javadocs

Which ones?

I think I'll address the notion of removal here along with what I said about other methods. I fully accept there is no harm in leaving these informational methods, so if we chose to keep them, we keep them! However at the same time, I still am not sure on what is particularly useful about them on a consumer level -- it's quite possible I'm missing something though.

I say this as my mindset is "make minimal, then expand" rather than provide things that end up being not useful and possibly changed down the road. What if Mojang changes these packs to have a numeric order, or `isLocked` is suddenly ignored. Now, I'm not sure that will happen, but erring on the side of caution is often the way I look at things.

The likelihood of this generally being an issue is low, but it's a consideration that is what leads me down to this suggestion. If it turns out someone will find it useful, we can always add it in a point release.